dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/dto/ProductInfoDto.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.api.dto;
+
+import lombok.Data;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * ProductInfoDto
+ */
```

Comment:
```suggestion
/**
 * ProductInfoDto
 */
```

Reply:
> 

My original idea for the throw code was to report errors when there is data but the version field is empty or null
Based on your proposal, I have made the following changes:
1. Remove error prompts and directly display unknown when missing data
2. Modify API URL path
3. Remove method comments
![image](https://github.com/apache/dolphinscheduler/assets/45991723/86b1f9f8-b3cb-4fbe-bd78-e94292f2f708)

Reply:
This comment is meanless, you need to remove.